pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -146,14 +147,15 @@ in stdenv.mkDerivation (fBuildAttrs // {
   preConfigure = ''
     mkdir -p "$bazelOut"
 
-    test "${bazel.name}" = "$(<$deps/.nix-bazel-version)" || {
+    (cd $bazelOut && tar xfz $deps)
```

Comment:
> Note you'll have to manually trigger the builds for the derivations that use buildBazelPackage to make sure they all build correctly.

Unlike normal FOD staleness scenarios, this one happens to be safe, because if you forget to change the FOD and end up with a stale hash, this line will fail in the builder, since your input will be a directory that can't be extracted with `tar`.